.travis.yml

```diff
@@ -49,18 +49,19 @@ before_install:
       git clone \
         https://${GH_APIDOC_TOKEN}@github.com/awesomeWM/awesomeWM.github.io \
         "$BUILD_WEB" 2>&1 | sed "s/$GH_APIDOC_TOKEN/GH_APIDOC_TOKEN/g"
-      if [ "$TRAVIS_PULL_REQUEST" = false ]; then
-        branch="$TRAVIS_BRANCH"
-      else
-        branch="pr-$TRAVIS_PULL_REQUEST"
-      fi
-      if [ "$branch" != master ]; then
-        cd $BUILD_WEB
-        if ! git checkout -b "$branch" "origin/$branch"; then
-          git checkout -b "$branch"
+      (
+        cd "$BUILD_WEB"
```

Comment:
but it's still `cd`, not `--git-dir/--work-tree/-C/whatever`

Reply:
It's in a subshell now.